src/admin/src/components/composite/advisoryForm/AdvisoryForm.js

```diff
@@ -208,15 +208,28 @@ export default function AdvisoryForm({
     required: false,
   };
 
-  const displayedDateOptions = (mode === "update") ? [
+  const displayedDateOptions = [
     { label: "Posting date", value: "posting" },
-    ({ label: "Updated date", value: "updated" }),
-    { label: "No dates", value: "no" }
-  ] : [
-    { label: "Posting date", value: "posting" },
-    { label: "No dates", value: "no" }
+    ...(mode === "update" ? [{ label: "Updated date", value: "updated" }] : []),
+    { label: "Start date", value: "start" },
+    { label: "Event date range", value: "event" },
+    { label: "No date", value: "no" }
   ];
 
+  const getDisplayedDate = () => {
+    if (!displayStartDate && !displayEndDate && !displayAdvisoryDate && !displayUpdatedDate) {
+      return displayedDateOptions[4];
```

Comment:
Can you declare constants for the array index values 0, 1, 2, 3, 4 to make this more readable?
e.g. `const POSTING_DATE = 3`